src/ssl.c

```diff
@@ -34816,7 +30693,7 @@ static int SetECPointInternal(WOLFSSL_EC_POINT *p)
 #endif /* HAVE_ECC */
 #endif /* OPENSSL_EXTRA */
 
-#if defined(HAVE_ECC) && defined(OPENSSL_EXTRA_X509_SMALL)
+#if defined(HAVE_ECC) && defined(OPENSSL_EXTRA)
```

Comment:
This used to be a feature that was a subset of OPENSSL_EXTRA code. For a use case that was looking for a subset of X509 operations pulled from the OPENSSL_EXTRA build.